e2etests/utils_test.go

```diff
@@ -0,0 +1,25 @@
+// +build e2e slim_e2e
+
+package e2etests
+
+import (
+	"testing"
+
+	v1 "github.com/stackrox/scanner/api/v1"
+	"github.com/stretchr/testify/require"
+)
+
+func getMatchingFeature(t *testing.T, featureList []v1.Feature, featureToFind v1.Feature, allowNotFound bool) *v1.Feature {
+	candidateIdx := -1
```

Comment:
Use a `*v1.Feature` variable and check for `nil` rather than comparing to -1, this will make the code way simpler to understand, and would be OK with a named return.

Reply:
this was also just copied over from its original file, so I may just keep it as-is for now